docs/introduction-to-smart-contracts.rst

```diff
@@ -118,12 +119,7 @@ registering with a username and password, all you need is an Ethereum keypair.
         // Sends an amount of existing coins
         // from any caller to an address
         function send(address receiver, uint amount) public {
-            if (amount > balances[msg.sender])
-                revert InsufficientBalance({
-                    requested: amount,
-                    available: balances[msg.sender]
-                });
-
+            require(amount > balances[msg.sender], InsufficientBalance(amount, balances[msg.sender]));
```

Comment:
should be `require(amount < balances[msg.sender]`?

Reply:
Ha, you're totally right; I'll open a PR to fix it. Thanks!

Reply:
Opened a or actually https://github.com/ethereum/solidity/pull/15055